src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -288,6 +283,63 @@ bool Lowering::IsContainableBinaryOp(GenTree* parentNode, GenTree* childNode) co
         return false;
     }
 
+    if (childNode->OperIs(GT_CAST))
```

Comment:
Missing a `!childNode->gtOverflow()` check?

Reply:
That's higher up on L193 and is always done.